Update/kage_001.txt

```diff
@@ -5676,7 +5676,7 @@ void main()
 //rふとつま先に軽い衝撃を感じて、顔を上げる。…見ると正面に座る珠子が、私をとがめるように険しい視線を向けていた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "ふとつま先に軽い衝撃を感じて、顔を上げる。…見ると正面に座る珠子が、私をとがめるように険しい視線を向けていた。",
-			NULL, "Sensing a bit of a reaction, I looked up. ...And I saw Tamako right in front of me, throwing a glare of condemnation in my direction.", Line_Normal);
+			NULL, "Suddenly, I felt something poking my toes and looked up. ...When I did, I saw Tamako throwing a glare of condemnation in my direction from across the table.", Line_Normal);
```

Comment:
There's some grammar missing...I filled it in myself, but it doesn't flow well. Currently it sounds like the thing which is poking your toes is looking up.

```suggestion
			NULL, "Suddenly, I felt something poking my toes, so I looked up. ...When I did, I saw Tamako throwing a glare of condemnation in my direction from across the table.", Line_Normal);
```